examples/pylint/W0232_no_init.py

```diff
@@ -9,3 +9,5 @@ def does_more_stuff(self, n, m):
 class ClassWithNoInitTwo(ClassWithNoInit):
```

Comment:
@WChrisK You can delete this check (please add it to the disable list, and remove it from the website documentation as well).